src/main/flight/mixer.c

```diff
@@ -765,6 +765,10 @@ void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensation)
     }
 
     motorMixRange = motorMixMax - motorMixMin;
+    if( throttleBoost > 0.0f) {
+        float throttlehpf = throttle - pt1FilterApply( &throttleLpf, throttle );
+        throttle = constrainf( throttle + throttleBoost * throttlehpf, 0.0f, 1.0f );
```

Comment:
Betaflight coding style is to not have spaces directly on the inside of braces (i.e. `throttle = constrainf(throttle + throttleBoost * throttlehpf, 0.0f, 1.0f)`). Can you please change this?

Reply:
done